app/controllers/forms/check_your_answers_controller.rb

```diff
@@ -17,13 +17,19 @@ def submit_answers
         if current_context.form_submitted?
           redirect_to error_repeat_submission_path(current_form.id)
         else
+          submission_id = SecureRandom.alphanumeric(8).upcase
```

Comment:
With Tom's work on [generating external IDs](https://github.com/alphagov/forms-admin/blob/af1c4c72bf96abd112e87bc7d9e0894967105b87/app/lib/external_id_provider.rb#L3), it made sense to use SecureRandom.base58. This was because it strips out characters that could be confused with each other, and I think in this case it would also make sense to avoid the I/l and 0/O (https://api.rubyonrails.org/classes/SecureRandom.html) in case they get mixed up. I think we could replace this with something like 

```suggestion
          submission_id = SecureRandom.base58(8).upcase
```

Reply:
Ooh, great shout - didn't know that existed but should be really handy in case form fillers have to quote their reference over the phone or want to write it down.

Reply:
Done!